src/solvers/postSolver.ml

```diff
@@ -211,7 +211,7 @@ struct
       if M.tracing then M.trace "postsolver" "one_constraint %a %a\n" S.Var.pretty_trace x S.Dom.pretty rhs;
       PS.one_constraint ~vh ~x ~rhs
     in
-    List.iter one_var vs;
+    (Stats.time "potentially_incremental_reach" (List.iter one_var)) vs;
```

Comment:
Is there any point to this timing any more since it's inside `postsolver` timing anyway? Or at least the name should probably be something different now, like just `iter`?

Reply:
I renamed it.